json-guides/retry-timeout.json

```diff
@@ -481,7 +481,9 @@
             "name": "WhatNext",
             "title": "Great work!  You're done!",
             "description": [
-            ]
+                "You learned about the benefits of Retry and Timeout policies and how to make your microservice more resilient to failures. You learned that the Timeout policy can be used to quickly fail requests that are hanging for too long. You learned to use the Retry policy to re-attempt an operation that has failed and therefore recovered from unpredictable failures your application may experience.",
```

Comment:
Change " .... recovered from unpredictable failures....."
to "....recover from transient failures...."